hooks/src/index.js

```diff
@@ -55,27 +85,8 @@ options._render = vnode => {
 	previousComponent = currentComponent;
 };
 
-options.diffed = vnode => {
-	if (oldAfterDiff) oldAfterDiff(vnode);
-
-	const c = vnode._component;
-	if (c && c.__hooks) {
-		if (c.__hooks._pendingEffects.length) afterPaint(afterPaintEffects.push(c));
-		c.__hooks._list.forEach(hookItem => {
-			if (hookItem._pendingArgs) {
-				hookItem._args = hookItem._pendingArgs;
-			}
-			if (hookItem._pendingValue !== EMPTY) {
-				hookItem._value = hookItem._pendingValue;
-			}
-			hookItem._pendingArgs = undefined;
-			hookItem._pendingValue = EMPTY;
-		});
-	}
-	previousComponent = currentComponent = null;
-};
-
 options._commit = (vnode, commitQueue) => {
+	mask = 0;
```

Comment:
Currently this is the reason why we store `_mask` on vnodes, ideally we introduce a better way to reset this on the server-side. We could offer this as an `options` attribute, hwoever that would mean we need to cut a new `renderToString` and `preact-ssr-prepass` minor